docs/book/src/topics/clusterclass.md

```diff
@@ -0,0 +1,79 @@
+# ClusterClass
+
+- **Feature status:** GA
```

Comment:
ClusterClass isn't GA, it requires a feature gate?

Reply:
oops!